src/components/shared/Avatar/Avatar.tsx

```diff
@@ -34,15 +34,15 @@ const displayName = 'Avatar';
 
 const Avatar = ({
```

Comment:
Please make sure your changes are more-or-less compatible with @Julianahlk 's changes in #78